docs/source/claimed_systems.rst

```diff
@@ -0,0 +1,201 @@
+Benchmarking Systems
+####################
+
+Here, we provide the full list of benchmarking systems from the `Schrodinger binding free energy benchmarks <https://github.com/schrodinger/public_binding_free_energy_benchmark/tree/v2.0>`_.
+The claimed systems are the datasets that have been picked by our industrial partners to be prepared and run, while the unclaimed systems will not be included in the benchmarking effort.
+We will update this list if more systems are being claimed.
```

Comment:
@jameseastwood do we want to advertise folks picking up more systems, as in "if you are interested in running any of these systems, please get in touch with the openfe team"?

Reply:
I will do this when we announce the repo.